handler/bot.go

```diff
@@ -0,0 +1,51 @@
+package handler
+
+import (
+	"log"
+	"time"
+
+	"github.com/ichiro-its/discord-pr-bot/config"
+	"github.com/ichiro-its/discord-pr-bot/service"
+	"github.com/ichiro-its/discord-pr-bot/service/impl"
+)
+
+type Bot struct {
+	discordService service.DiscordService
+	githubService  service.GithubService
+	channelID      string
+	messageID      string
+}
+
+func NewBot(config *config.Config) *Bot {
+	return &Bot{
+		discordService: impl.NewDiscordService(config.DiscordBotToken),
+		githubService:  impl.NewGithubService(),
+		channelID:      config.DiscordChannelID,
+		messageID:      config.DiscordMessageID,
+	}
+}
+
+func (b *Bot) Process() {
+	// Get open pull request urls
+	pullRequestUrls, err := b.githubService.GetOpenPullRequestUrls()
+	if err != nil {
+		log.Panicf("Failed to get open pull request urls: %+v", err)
+	}
```

Comment:
In my opinion, it's better to just return an error object in here, and let the panic to be handled by the caller.  This may simplify the testing.

Reply:
this function currently wraps the whole flow and doesn't return an error because I want to keep the `main` as simple as possible

Do you think we should change?

Reply:
It's up to you to decide

Reply:
hmm okay, I think can do panic in the `main` instead